packages/grafana-ui/src/components/Forms/Button.tsx

```diff
@@ -127,7 +134,7 @@ export const getButtonStyles = stylesFactory(({ theme, size, variant }: StylePro
 });
 
 // These are different from the standard Button where there are more variants.
-export type ButtonVariant = 'primary' | 'secondary' | 'destructive' | 'link';
+export type ButtonVariant = 'primary' | 'secondary' | 'destructive' | 'link' | 'primary-legacy';
```

Comment:
Does this mean are we adding `legacy` styles to new Form styles?

Reply:
🤦‍♂ need to remove this. Good catch